security-admin/src/main/java/org/apache/ranger/rest/PublicAPIs.java

```diff
@@ -415,4 +417,44 @@ public RangerManagementConfig getManagementConfiguration(@Context HttpServletReq
 		return rangerManagementConfig;
 	}
 
+	@GET
+	@Path("/admin/ok")
+	public String checkRangerHealth (@Context HttpServletRequest request) {
+
+		if (logger.isDebugEnabled()) {
+			logger.debug("==> PublicAPIs.checkRangerHealth(): ");
+		}
+		HashMap<String, String> responseMap = new HashMap<>();
+		JSONUtil jsonUtil = new JSONUtil();
+
+		// Default Response Parameters
+		String serviceStatus = "partially running";
+		String dbStatus = "not running";
+		String version = RangerVersionInfo.getVersion();
+		boolean isSuccess = false;
+
+		// Get Database Version
+		String dbVersion = daoMgr.getXXDataHist().getDBVersion();
+
+		if (!dbVersion.equalsIgnoreCase("Not Available")) {
```

Comment:
you'll need to make NotAvailable constant in:
 https://github.com/Altiscale/ranger/blob/branch-0.7.1-alti/security-admin/src/main/java/org/apache/ranger/common/db/BaseDao.java#L254

Then it'll be safe to use it as an indication that everything else failed. 

Other than that, looks good to me.